src/auth/frappe.auth.controller.ts

```diff
@@ -80,15 +80,29 @@ export default class FrappeAuthController extends AuthController {
         break;
 
       case "pin_login":
-        err = {
-          ...error,
-          title: "Incorrect Pin",
-          info: {
-            ...error.info,
-            cause: "Wrong PIN is entered",
-            suggestion: "Re-enter the PIN correctly"
-          }
-        };
+        switch (error.info.data.message) {
```

Comment:
I think it's better to keep it as if/else since the check may not be `error.info.data.message`

Reply:
yep 👍